src/Raven.Server/Config/Categories/ReplicationConfiguration.cs

```diff
@@ -42,5 +42,11 @@ public class ReplicationConfiguration : ConfigurationCategory
         [SizeUnit(SizeUnit.Megabytes)]
         [ConfigurationEntry("Replication.MaxSizeToSendInMb", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public Size? MaxSizeToSend { get; set; }
+
+        [Description("Maximum number of data size to load from storage to memory before sending it")]
+        [DefaultValue(1024)]
+        [SizeUnit(SizeUnit.Megabytes)]
+        [ConfigurationEntry("Replication.MaxSizeToLoadFromStorage", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        public Size? MaxSizeToLoadFromStorage { get; set; }
```

Comment:
Can we make it `128MB` for 32bit?

Reply:
The default value can be only a const value (not a variable), so I can do it 128mb for 32bit but it would be the same for 64bit..

Reply:
I don't follow about the same comment?

Reply:
That I have he same comment as here: https://github.com/ravendb/ravendb/pull/15286#discussion_r1030566082